localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```diff
@@ -38,7 +38,7 @@
  innovation_candleholder:0 "Candleholders"
  innovation_candleholder_desc:0 "You no take candle! We protect the Candle King."
 
- innovation_cloud_serpents:0 "Cloud Serpents"
+ innovation_cloud_serpents:0 "Order of the Cloud Serpents"
```

Comment:
I'd rather go with "Cloud Serpent taming" or something like this. If they're a order, they'd rather either hold land (like a holy order) or be a mercenary group.

Reply:
https://wowwiki-archive.fandom.com/wiki/Order_of_the_Cloud_Serpent

I left "xxx taming" for regional techs

Reply:
Uuuhm, since this order was founded during the Zandalari War, why make it a special innovation at all? Wouldn't they be unlocked with all the other pandaren MaAs from the start? like we have for elven innovations?

Reply:
Other maas are unlocked later

Reply:
And why? Did they have some sort of evolution/tech revolution that made them have bearpaws and stuff?

Reply:
not being enslaved, idk.

Reply:
This question was reasked below, I will resolve this and move the conversation down.